### ISSUE TITLE
put Cyberark onto Access Boston Portal

### ISSUE THREAD

**Issue by stephaniemar** [CityOfBoston/boston.gov-d8#2329]
timeframe for rollout is March 1
Small set of internal (DoIT) users so can be anytime of the day
Icon done already (another lock)
Will get you the Prod URL shortly
And the security group who will see this icon

**Comment by stephaniemar** [comment]
Hey @davidrkupton and @duffy-james -- New request from Access Boston to review and prioritize. Looks like they have a March 1 date for going live.

**Comment by davidrkupton** [comment]
OK thanks stephanie - Should this ticket be in the digital repo? Access Boston is an  app in that repo.

**Comment by stephaniemar** [comment]
Yeah, I was not sure which repo to attach it to, I think i can update the repo
